Sources/MapboxMaps/Foundation/MapViewDependencyProvider.swift

```diff
@@ -162,9 +165,14 @@ internal final class MapViewDependencyProvider: MapViewDependencyProviderProtoco
             puckManager: puckManager)
     }
 
-    func makeViewportImpl(mapboxMap: MapboxMapProtocol,
+    func makeViewportImpl(view: UIView,
+                          mapboxMap: MapboxMapProtocol,
                           cameraAnimationsManager: CameraAnimationsManagerProtocol,
-                          idleGestureRecognizer: UIGestureRecognizer) -> ViewportImplProtocol {
+                          idleGestureRecognizers: [UIGestureRecognizer]) -> ViewportImplProtocol {
+        let gestureRecognizer = DiscreteAnyTouchGestureRecognizer(
+            minimumPressDuration: 0.15,
```

Comment:
Following refactoring, it's no longer repeated in multiple places. I actually prefer it inline since it lets you see the context of how it's used without searching.

I will add a comment to describe why I picked 0.15 though.